working-open.md

```diff
@@ -0,0 +1,53 @@
+# Working Open
+
+Hypha operates with democratic organizing principles informed by our [values](./values.md). As part of that, we've developed a set of guidelines for how (and exactly what) and why we work in the open.
+
+## Why?
+
+- In order to share and provide opportunities for self- and collective- learning as we work in solidarity with others seeking to create meaningful livelihoods.
+- Not because "default to open" is an ideal in itself, but because it orients us toward negotiating a set of practices around our shared intentions.
+- Yet we recognize openly sharing might prevent frank and necessary discussion beneficial to our members, and so we want to clearly lay out our practices of consent and privacy
+
+# How and what?
+
+- We currently work with an [*in camera*](https://en.wikipedia.org/wiki/In_camera)/redaction request for meeting notes, where at any point members can request to stop taking notes, or redact and correct notes after they have been taken
```

Comment:
Re-reading. Perhaps we could strike the use of "in camera"? It's a historic but perpetually-confusing term that means the opposite of what contemporary people assume. Here's an attempt to trim/simplify.

```suggestion
- For meeting notes, we accommodate redactions and going off-the-record. Participants can request to stop taking notes, or redact and correct notes after they have been taken.
```

Reply:
Members => participants, because this isn't just for members, right?

Reply:
I'm not sure about other folks, but if anyone has attended a more formally chaired meeting than an in camera process is not likely unfamiliar. I'm fine with clarifying language, I guess I wonder if we didn't already address your concerns with the following statements?

I'd also prefer we keep the temporal aspects: that this is our current practice.